xmr-btc/tests/harness/mod.rs

```diff
@@ -0,0 +1,36 @@
+pub mod node;
+pub mod transport;
+pub mod wallet;
+
+pub mod bob {
+    use xmr_btc::bob::State;
+
+    // TODO: use macro or generics
+    pub fn is_state5(state: &State) -> bool {
+        matches!(state, State::State5 { .. })
```

Comment:
:upside_down_face: Given that it's test code we could just inline these and reduce lines of code.

Reply:
i tried inlining the matches! macro but the run_until function need a function as an argument which takes in the state which the execution has progressed to.